### PR TITLE
my-repo-1 commit 2

### PR DESCRIPTION
**Pull request by sagar-ab-2702**

### CODE REVIEW

File: file-2.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+this is file 2
```
Comment (sagar-ab-2702) [inline file-2.txt:1]: Fix this

Comment (sagar-ab-2702) [inline file-2.txt:1]: Fix

Comment (sagar-ab-2702) [inline file-2.txt:1]: qw

Comment (AditiSharma13) [reply file-2.txt:1]: okay will fix

**Review by sagar-ab-2702 — COMMENTED** [review]
Submit revew

**Pull request merged**